etc/kayobe/ansible/scripts/overcloud-ubuntu-upgrade.sh

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/bash
+
+set -e
+
+if [[ ! $1 ]]; then
+    echo "Usage: overcloud-ubuntu-upgrade.sh <overcloud-hostname>"
+    exit 2
+fi
+
+if [[ ! $KAYOBE_PATH ]]; then
+    echo "Environment variable \$KAYOBE_PATH is not defined"
+    exit 2
+fi
+
+if [[ ! $KAYOBE_CONFIG_PATH ]]; then
+    echo "Environment variable \$KAYOBE_CONFIG_PATH is not defined"
+    exit 2
+fi
+
+if [[ ! $ANSIBLE_ROLES_PATH ]]; then
+    set -x
+    export ANSIBLE_ROLES_PATH=$KAYOBE_PATH/ansible/roles
+    set +x
+else
+    set -x
+    export ANSIBLE_ROLES_PATH=$ANSIBLE_ROLES_PATH:$KAYOBE_PATH/ansible/roles
+    set +x
+fi
+
+set -x
+
+kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/ubuntu-upgrade.yml --limit $1
+
+kayobe overcloud host configure --limit $1
```

Comment:
```suggestion
kayobe overcloud host configure --limit $1 --kolla-limit $1
```

Reply:
If you are on antelope when running that command then you will hit an error because `--kolla-limit` is no longer used with `host configure`. Although I'm guessing that this will probably not be the case and most likely will be on yoga?

Reply:
Added those changes